chromium_src/components/content_settings/browser/content_settings_manager_impl.cc

```diff
@@ -0,0 +1,57 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "net/cookies/static_cookie_policy.h"
+#include "third_party/blink/public/common/features.h"
+
+#define BRAVE_ALLOW_STORAGE_ACCESS \
+  CheckEphemeralStorageAccess(url, site_for_cookies, storage_type, allowed);
+
+#include "../../../../../components/content_settings/browser/content_settings_manager_impl.cc"
+#undef BRAVE_ALLOW_STORAGE_ACCESS
+
+namespace content_settings {
+namespace {
+
+bool isEphemeralStorageOn() {
```

Comment:
It's probably better to move this into the body of the method below since you only it once.

Reply:
Done